maxim/tests/test_logger_langchain_03x.py

```
@@ -164,279 +159,279 @@ def test_generation_chat_prompt_azure_chat_model_with_streaming(self):
+    #         region="us-east-1",
+    #         aws_access_key_id=awsAccessKeyId,
+    #         aws_secret_access_key=awsAccessKeySecret,
+    #         model_kwargs={"temperature": 0.1},
+    #         provider="anthropic",
+    #         model="anthropic.claude-3-5-sonnet-20240620-v1:0",
+    #         callbacks=[MaximLangchainTracer(logger)],
+    #     )
+    #     messages = [
+    #         (
+    #             "system",
+    #             "You are a helpful assistant that translates English to French. Translate the user sentence.",
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)